lib/momentum/momentum.h

```diff
@@ -88,9 +96,9 @@ typedef struct {
     FuriHalSerialId uart_nmea_channel;
     bool file_naming_prefix_after;
     VgmColorMode vgm_color_mode;
-    Rgb565Color vgm_color_fg;
-    Rgb565Color vgm_color_bg;
     FuriHalVersionColor spoof_color;
+    RgbColorTransmit vgm_color_fg;
+    RgbColorTransmit vgm_color_bg;
```

Comment:
changing data type here, might be good to change name too so it is not imported incorrectly. then again, also need to update the code part in settings.c, so maybe can change it to save as hex and check data length instead of saving as uint32